rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -450,3 +451,44 @@ def _list_oflow_count_for_sandboxes(self, sandboxes,
             "chart_plugin": "StackedArea", "data": oflow_data
         }
         self.add_output(additive_oflow_data)
+
+    def _create_address_set(self, set_name, address_list):
+        LOG.info("create %s address_set [%s]" % (set_name, address_list))
+
+        name = "name=\"" + set_name + "\""
+        addr_list="\"" + address_list + "\""
+
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ovn_nbctl.create("Address_Set", name, ('addresses', addr_list))
+        ovn_nbctl.flush()
+
+    def _modify_address_set(self, set_name, address_list):
+        LOG.info("modify %s address_set [%s]" % (set_name, address_list))
+
+        name = "\"" + set_name + "\""
+        addr_list="\"" + address_list + "\""
+
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ovn_nbctl.set("Address_Set", name, ('addresses', addr_list))
+        ovn_nbctl.flush()
+
+    def _remove_address_set(self, set_name):
+        LOG.info("remove %s address_set" % set_name)
+
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ovn_nbctl.destroy("Address_Set", set_name)
+        ovn_nbctl.flush()
+
+    def _get_address_set(self, set_name):
+        LOG.info("get %s address_set" % set_name)
+
+        name = "\"" + set_name + "\""
+        ovn_nbctl = self.controller_client("ovn-nbctl")
+        ovn_nbctl.set_sandbox("controller-sandbox", self.install_method)
+        ret = ovn_nbctl.get("Address_Set", name, ['addresses'])
+        ovn_nbctl.flush()
```

Comment:
One more problem here. I don't think flush() can help for get(). For the get() to return result, we must set batch mode to false before calling the get(). And the flush() can be removed.

Reply:
as suggested by @jsitnicki I will add the 'add' routine to DdCtlMixin class in order to update the address_set. Do you think it is worth to have 'get' routine as well or do you prefer to remove it?

Reply:
get() is one of the standard operation, so it is valuable. If it is not needed for this patch it is ok to add it in a separate patch.